src/AuthorProcessor.php

```diff
@@ -0,0 +1,223 @@
+<?php 
+
+/*
+ * This file is part of the BibTex Parser. This file was contributed by Andre Chalom <andrechalom@gmail.com>.
+ *
+ * BibTex Parser is (c) Renan de Lima Barbosa <renandelima@gmail.com>
+ *
+ * This file includes source code adapted from the Structures_BibTex package, (c) Elmar Pitschke <elmar.pitschke@gmx.de>,
+ * included here under PHP license: http://www.php.net/license/3_0.txt.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace RenanBr\BibTexParser;
+
+class AuthorProcessor {
```

Comment:
All right. So the proper way to do it is writing a comment just before the class definition, and not in the file header?